### PR TITLE
fix: Workaround for Posts page

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [x] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Description

Now we can use the edit and delete buttons on Android in the posts page.

See https://github.com/unoplatform/uno/issues/13996 for more details.

## Impact on version

- [ ] **Major**
  - The template structure was changed.
- [ ] **Minor**
  - New functionalities were added.
- [x] **Patch**
  - A bug in behavior was fixed.
  - Documentation was changed.

### Always applicable
No matter your changes, these checks always apply.
- [x] Your conventional commits are aligned with the **Impact on version** section.
- [x] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [x] Documentation files were updated according with the changes.
  - Update `README.md` and `TemplateConfig.md` if you made changes to **templating**.
  - Update `AzurePipelines.md` and `APP_README.md` if you made changes to **pipelines**.
  - Update `Diagnostics.md` if you made changes to **diagnostic tools**.
  - Update `Architecture.md` and its diagrams if you made **architecture decisions** or if you introduced new **recipes**.
  - ...and so forth: Make sure you update the documentation files associated to the recipes you changed. Review the topics by looking at the content of the `doc/` folder.
- [x] Images about the template are referenced from the [wiki](https://github.com/nventive/UnoApplicationTemplate/wiki/Images) and added as images in this git.
- [x] TODO comments are hints for next steps for users of the template and not planned work.

### Contextual
Based on your changes these checks may not apply.
- [x] Automated tests for the changes have been added/updated.
- [x] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Access/ApplicationTemplate.Access.csproj [hunk 1]
```diff
@@ -24,9 +24,13 @@
...
 		<PackageReference Include="System.Text.Json" Version="7.0.3" />
-		<PackageReference Include="MallardMessageHandlers" Version="0.2.0-dev.21" />
+		<PackageReference Include="MallardMessageHandlers" Version="1.2.0" />
 		<PackageReference Include="Nventive.Persistence.Reactive" Version="0.5.0" />
 		<PackageReference Include="Nventive.Persistence" Version="0.5.0" />
 		<PackageReference Include="System.Reactive" Version="5.0.0" />
 	</ItemGroup>
+
+	<ItemGroup>
+	  <PackageReference Update="GooseAnalyzers" Version="1.0.0" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Access/ApplicationTemplate.Access.csproj:34]: Don't add a second reference to GooseAnalyzers. (It's already referenced in Directory.Build.props)

File: src/app/ApplicationTemplate.Shared.Views/Content/Posts/PostsPage.xaml [hunk 2]
```diff
@@ -75,20 +66,17 @@
...
 			<!-- Create New Post Button -->
 			<Button Command="{Binding NavigateToNewPost}"
-					CommandParameter="{Binding}"
 					Style="{StaticResource IconButtonStyle}"
 					Background="Transparent"
 					HorizontalAlignment="Right"
 					VerticalAlignment="Bottom"
 					Margin="24">
 
-				<!-- Border -->
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Shared.Views/Content/Posts/PostsPage.xaml:85]: No need to remove this

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

**Comment by juliecantin** [comment]
Please consolidate into a single commit

**Comment by Soap-141** [comment]
> Please consolidate into a single commit

@juliecantin Done.

**Review by takla21 — APPROVED** [review]

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**